pyproject.toml

```
@@ -33,7 +33,8 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Artificial Life",
 ]
 dependencies = [
-  "numpy~=1.26",
+  "numpy~=2.0.2",
+  "numba",
```

```suggestion
  "numpy>=2.0.2",
```

This will make sure the latest compatible version of numpy is installed.
Also numba is already in the dev dependencies so we can delete it